### DISCUSSION TITLE
Logging outside container?

### DISCUSSION THREAD

**Discussion by stevenbaert** [open-webui/open-webui#4774] [Q&A]
Hi,

I try to log to a file outside the open-webui container but I fail to do so.
Below the important parts  of my yaml file. Container is created succesfully, but there is no logfile being created.

Thanks for your input!
S.

    image: ghcr.io/open-webui/open-webui:latest

      - LOG_FILE=/app/backend/logs/open-webui.log
      - LOG_LEVEL=DEBUG
    ports:
      - 9090:8080
    volumes:
      - /volume1/docker/open-webui/chat/data/:/app/backend/data
      - /volume1/docker/open-webui/chat/logs/:/app/backend/logs

**Comment by thiswillbeyourgithub** [comment]
Personnaly I just logged the container to systemd: 
```
    logging:
      driver: journald
```

The logging is not specific to openwebui so instead take a look at [docker logging drivers](https://docs.docker.com/engine/logging/configure/)

**Reply by stevenbaert** [reply]
No easier way than that? Bummer.
Haven't got the time right now to figure this out. Was hoping for easy redirection via container variables.
